inception-project/inception-project.github.io
# #55 INCEpTION 0.16.1

**PR contains**
- update releases
- add docu

**Test manually**
- build the webpage
- check that jar can be downloaded and guides are current (0.16.1)
